chatez/src/pages/Chats/Components/ChatFeed/ChatFeed.tsx

```diff
@@ -265,9 +296,21 @@ const ChatFeed: React.FC = () => {
 
         {/* Message Input Area */}
         <div className="Bottom-panel">
-          <IconButton id="Image-button" onClick={insertImage}>
-            <Image id="Image-icon" />
-          </IconButton>
+          {attachedFile && showAttachmentPreview && (
+            <div className="attachment-preview">
+              {['image/png', 'image/jpeg', 'image/gif'].includes(attachedFile.type) ? (
```

Comment:
this is a bit nit picky but can you put this in a list and then use it here. it would just be easier to edit it up there then scroll all the down here. give it an appropriate name as well. pls and ty